backend/core/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "declaration": true,
+    "removeComments": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "target": "es2017",
+    "sourceMap": true,
+    "outDir": "./dist",
+    "baseUrl": "./",
+    "incremental": true,
+    "esModuleInterop": true,
+    "resolveJsonModule": true,
```

Comment:
Not sure if this matters, but it's missing a couple keys that are in the root `tsconfig.json` for the mono repo:

`isolatedModules`, `forceConsistentCasingInFileNames`, `skipLibCheck`, `strict: false`.

There might be good reasons to have module-specific tsconfigs, but I think we probably want to keep them as consistent as possible, no?

Reply:
Will try adding them and see how it plays out.